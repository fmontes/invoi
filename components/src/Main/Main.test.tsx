import * as React from 'react';
import { shallow } from 'enzyme';
import { Layout } from './';

test('Layout should render', () => {
    const layout = shallow(<Layout><h2>Hello World</h2></Layout>);
    expect(layout).toMatchSnapshot();
});
