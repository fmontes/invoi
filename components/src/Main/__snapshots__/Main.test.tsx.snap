// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render 1`] = `
<styled.main>
  <h2>
    Hello World
  </h2>
</styled.main>
`;
